src/components/layouts/CgPosthogProvider.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+
+import config from "@/config";
+import posthog from "posthog-js";
+import { PostHogProvider } from "posthog-js/react";
+
+if (typeof window !== "undefined") {
+  posthog.init(config.posthogApiKey, {
+    api_host: "/ingest",
+    ui_host: "https://us.posthog.com",
+    autocapture: {
+      dom_event_allowlist: ["click"],
+      element_allowlist: ["button", "a"],
+    },
```

Comment:
whats the default?

Reply:
I think turning on auto capture in config on dashboard activates it, but specifying where you want to auto capture is why we have the arguments passed.

Reply:
we dont have autocapture in other projects and we see the events of on click

Reply:
<img width="677" alt="Screenshot 2024-06-23 at 5 02 25 PM" src="https://github.com/EveripediaNetwork/braindao-ui/assets/66301634/3ad40d08-6092-4025-8c1b-e49b5115a5de">

Auto-capture is activated by default in dashboard settings for the project. Adding it to the configuration helps streamline its operation, such as specifying where auto-capture should be active, using allow lists and ignore lists.

Reply:
lets remove this, i think we can play with this once we have some data